Makefile

```diff
@@ -14,15 +14,21 @@ gofmt:
 
 .PHONY: bin
 bin:
-	 go build -o ./out/chart-verifier main.go
+	cp -r ./config ./pkg/chartverifier/profiles
+	go build -o ./out/chart-verifier main.go
+	rm -rf ./pkg/chartverifier/profiles/config
```

Comment:
Perhaps a new target can be created

```Makefile
.PHONY: copy-config
copy-config:
        rm -fr ./pkg/chartverifier/profiles/config
        cp -r ./config ./pkg/chartverifier/profiles

.PHONY: bin
bin: copy-config
        go build -o ./out/chart-verifier main.go
```

Reply:
decided to move profiles to a directory under pkg so this change is no longer needed.